recipe/meta.yaml

```diff
@@ -44,6 +42,8 @@ requirements:
     - ujson >=1.35
     - dill >=0.2,<0.3
     - requests >=2.13.0,<3.0.0
+    - regex ==2017.4.5
```

Comment:
Even though this actually works, it may be better to match existing conda version `2017.04.05` (leading zeros).